logstash-core/build.gradle

```diff
@@ -68,8 +68,10 @@ task copyGemjar(type: Copy, dependsOn: [sourcesJar, copyRuntimeLibs]) {
 }
 
 task cleanGemjar {
-    delete fileTree(project.file('lib/jars/')) {
-        include '*.jar'
+    doLast {
```

Comment:
Note for reviewer:
if the deletion is kept in Gradle's configuration phase, then when we run any other task on the parent project, this piece is executed, resulting in undesired deletions. This happens because Gradle parent project is configured, so configure the subprojects and configuring `logstash-core` scatter the deletion. If I invoked the copyJdk to prepare for bundle it ends in a tar without the jars that composes Logstash core (logstash-core.jar)